docs/excel/custom-functions-debugging.md

```diff
@@ -1,18 +1,30 @@
 ---
-title: UI-less custom functions debugging
-description: Learn how to debug your Excel custom functions that don't use a task pane.
-ms.date: 05/31/2022
+title: Custom functions debugging
+description: Learn how to debug your Excel custom functions that don't use a shared runtime.
+ms.date: 06/09/2022
 ms.localizationpriority: medium
 ---
 
-# UI-less custom functions debugging
+# Custom functions debugging
 
-This article discusses debugging *only* for custom functions that don't use a task pane or other user interface elements (UI-less custom functions).
+This article discusses debugging only for custom functions that **don't use a [shared runtime](../develop/configure-your-add-in-to-use-a-shared-runtime.md)**. To debug custom functions add-ins that use a shared runtime, see [Configure your Office Add-in to use a shared JavaScript runtime: Debug](../develop/configure-your-add-in-to-use-a-shared-runtime.md#debug).
 
 [!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
 
 [!include[Shared runtime note](../includes/shared-runtime-note.md)]
 
+## Requirements
+
+This debugging process works only for custom functions that **don't use a shared runtime**. To create a custom function that doesn't use a shared runtime, follow the steps in the [Create custom functions in Excel](../tutorials/excel-tutorial-create-custom-functions.md) tutorial and install an **Excel Custom Functions Add-in project** with the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md).
```

Comment:
The goal of the "To create" sentence is to clarify what a non-shared runtime custom function is, so that a customer can determine which runtime they have and whether this article applies. In essence, I'm trying to say that if they've used yo office, they'll have non-shared runtime unless they used yo office and then followed the shared runtime tutorial. A non-shared runtime custom function isn't defined anywhere in this article and I think it's helpful.

Reply:
If the purpose is to clarify what a non-shared runtime custom function is, then let's word it more like a definition: "A custom function that doesn't use a shared runtime is ..." What we have now is an imperative mood sentence that seems to give directions for creating something. 

BTW, since we don't recommend creating custom functions that don't use a share runtime, we need to get Yo Office changed ASAP so that the custom functions option creates a project with a shared runtime.